client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -335,7 +335,7 @@ export const List = React.createClass( {
 
 export default connect( ( state, ownProps ) => {
 	return {
-		hasDomainCredit: hasDomainCredit( state, ownProps.selectedSite.ID )
+		hasDomainCredit: ownProps.selectedSite && hasDomainCredit( state, ownProps.selectedSite.ID )
```

Comment:
I'd prefer the latter of an explicit prop, but I added the former in 48a7bd2 for a quick fix